src/box/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/src/box/lua)
-file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/src/box/sql)
+file(MAKE_DIRECTORY ${PROJECT_BINARY_DIR}/src/box/lua)
```

Comment:
While we're here: why don't you use `${CMAKE_CURRENT_SOURCE_DIR}`/`${CMAKE_CURRENT_BINARY_DIR}` instead of `${PROJECT_SOURCE_DIR}/src/box`/`${PROJECT_BINARY_DIR}/src/box` within this CMakeLists.txt?

Reply:
I would keep my point of uniform use of `PROJECT_`, here's an incremental patch to consider: https://github.com/tarantool/tarantool/commit/8cbb4514dc59cf8757129d87d76262bc2842bbac
It doesn't make path shorter like
```
-    WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/src/box
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/box
```
nor brings clarity to the mechanics like
```
-    WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/src/box
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
```

Reply:
Now the patch is easy to read, while if we would use `PROJECT_` here and `CURRENT_` there, it would raise the question: what is the principle? I would leave it as is.

Reply:
Then OK, leave everything intact.